tests/cli/test_deploy_cmd.py

```diff
@@ -26,6 +26,40 @@ def test_deployment_command_with_one_specified_contract(project):
     # weak assertion but not sure what to do here.
     assert 'Deploying Math' in result.output
     assert 'Deploying WithNoArgumentConstructor' not in result.output
+    
+@load_contract_fixture('ImportTestD.sol')
```

Comment:
All of these deployment tests can be removed as the testing of import remappings should be isolated to the compilation tests.